packages/python-pypdf/PKGBUILD

```
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by mark.blakeney at bullet-systems dot net
+
+_name=pypdf
+pkgname=python-$_name
+pkgver=3.2.0
+pkgrel=1
+pkgdesc='Python library for manipulating pages of PDF files (v3+)'
+arch=(any)
+url="https://github.com/py-pdf/$_name"
+license=(BSD)
+depends=(python)
+makedepends=(python-setuptools python-build python-installer python-wheel python-flit-core)
+source=($pkgname-$pkgver.tar.gz::"$url/archive/$pkgver.tar.gz")
+sha256sums=('fc8ee0b1f680f88d61098daa3cc4657e06d0401f2eee797dc3e39081a8913d79')
+
+build() {
+  cd $_name-$pkgver
+  python -m build --wheel --no-isolation
+}
+
+package(){
+  cd $_name-$pkgver
+  python -m installer --destdir="$pkgdir" dist/*.whl
```

```suggestion
  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
```

Dos it work with this change?

---

No, pypdf source files don't have a `setup.py` (look https://github.com/py-pdf/pypdf)

---

I will use PKGBUILD-python-lib-PEP517 template